src/datepicker/datepicker.ts

```diff
@@ -94,7 +94,7 @@ export interface NgbDatepickerNavigateEvent {
       <ng-template ngFor let-month [ngForOf]="months" let-i="index">
         <div class="ngb-dp-month d-block ml-3">            
           <div *ngIf="navigation !== 'select' || displayMonths > 1" class="ngb-dp-month-name text-center">
-            {{ i18n.getMonthFullName(month.number) }} {{ month.year }}
+            {{ i18n.getMonthFullName(month.number) }} <span class="ngb-dp-year">{{ month.year }}</span>
```

Comment:
You can style the div however and then override the span as necessary.

But yes, if I was starting from scratch I would move the ngb-dp-month-name down to a span around the month name. But that could break existing styles and I didn't want to push it. If a new class is added I don't know what to name it, there's already ngb-dp-month and ngb-dp-month-name!

You could reuse a class and use the tag in the selector to choose but that's not very clean.

The cleanest ignoring compatibility would be to move ngb-dp-month-name down to the span and replace it with ngb-dp-month-year-text.